src/app/schemas/Class.js

```diff
@@ -0,0 +1,30 @@
+import mongoose from 'mongoose';
+
+const ClassSchema = new mongoose.Schema({
+	codigo: {
+		type: String,
+		required: true,
+	},
+	classe: {
+		type: String,
+		required: true,
+	},
+	teoria: {
+		type: String,
+	},
+	pratica: {
+		type: String,
+	},
+	docenteTeoria: {
+		type: String,
+	},
+	docentePratica: {
```

Comment:
I imagine these names are in Portuguese because they are in the PDF, right? But don't you think it makes sense to translate them before saving them in the bank and using them with the keys we've defined? So we are safe if prograd makes changes, as only the new app will use this API, we can develop it already considering the new key names too

Reply:
These key names are defined in the [classes generator program](https://github.com/centralufabc/gerador-aulas/blob/master/src/turmascentral/Generate.java#L135) and are not directly bonded to PROGRAD's PDF file.
Anyway, I will translate them to keep code consistency with everything in English. Thanks!